### ISSUE TITLE
Progress?

### ISSUE THREAD

**Issue by dojo-json** [CodingDojo-MEAN-04-18/Justin-Clark_MEAN1#1]
Any assignment updates?

**Comment by justnclrk** [comment]
Yes, sorry, I'm about 30% through the angular documentation on the
platform. I just pushed what I have going so far, i'm going to pick back up
in the morning with event-binding. Will a new exam code be out today?

**Comment by dojo-json** [comment]
I will be generating a new code and sending it out today.

Consider closing this issue with a commit message. More details here: https://help.github.com/articles/closing-issues-using-keywords/
